node/src/actors/json_rpc/json_rpc_methods.rs

```diff
@@ -534,17 +534,17 @@ mod tests {
             value: 0,
         });
         let reveal_output = Output::Reveal(RevealOutput {
-            pkh: [0; 20],
+            pkh: PublicKeyHash::default(),
             reveal: [0; 32].to_vec(),
             value: 0,
         });
         let consensus_output = Output::Tally(TallyOutput {
-            pkh: [0; 20],
+            pkh: PublicKeyHash::default(),
             result: [0; 32].to_vec(),
             value: 0,
         });
         let value_transfer_output = Output::ValueTransfer(ValueTransferOutput {
-            pkh: [0; 20],
+            pkh: PublicKeyHash::default(),
```

Comment:
In line 558 and 777 use transaction_example() to use a transaction with default fields